### DISCUSSION TITLE
"/static_distance_map_ref" and "/static_obstacle_map_ref" not published

### DISCUSSION THREAD

**Discussion by Yuichiro-HIROSE** [HumanSupportRobot/QandA#53] [Q&A] [answered]
Our HSR-B doesn't publish the topics in the title while those 3 topics are published.
- /dynamic_obstacle_map
- /static_distance_ros_map
- /static_distance_ros_map

Also, this gazebo does publish the all 5 map topics.
https://github.com/hsr-project/hsrb_wrs_gazebo_launch
HSR software version is 22.10.
Any ideas/suggestions to fix or investigate this would be appriciated!

**Comment by Yuichiro-HIROSE** [comment]
What "rostopic info" says in gazebo is this for both topics.
>Type: nav_msgs/OccupancyGrid
> Publishers: /base_rviz_wrapper

**Comment by arata** [comment, marked as answer]
We had the same problem with our HSRB-33. I also confirmed that it was published map topic "/static_distance_map_ref" and "/static_obstacle_map_ref" again after a clean install 22.10.
However, [HSR manual](https://docs.hsr.io/hsr_develop_manual/reference/changelogs/23_04.html) says that the changes regarding the map were made starting with version 23.04, so I am wondering why it suddenly stopped publishing map topic with 22.10.

**Reply by Yuichiro-HIROSE** [reply]
Thanks a lot! After clean install, it works. FYI, we followed this procedure for clean install.
https://docs.hsr.io/hsrb_user_manual_en/howto/clean_install.html#id7
